src/controllers/DrawingModeController.js

```diff
@@ -52,47 +46,25 @@ class DrawingModeController extends AnnotationModeController {
             this.setupHeader(this.container, shell);
         }
 
-        this.cancelButtonEl = this.getButton(SELECTOR_ANNOTATION_BUTTON_DRAW_CANCEL);
-        this.postButtonEl = this.getButton(SELECTOR_ANNOTATION_BUTTON_DRAW_POST);
-        this.undoButtonEl = this.getButton(SELECTOR_ANNOTATION_BUTTON_DRAW_UNDO);
-        this.redoButtonEl = this.getButton(SELECTOR_ANNOTATION_BUTTON_DRAW_REDO);
-
         this.handleSelection = this.handleSelection.bind(this);
     }
 
     /**
-     * Disables the specified annotation mode
-     *
      * @inheritdoc
-     * @return {void}
      */
-    exit() {
-        this.emit(CONTROLLER_EVENT.exit, { headerSelector: SELECTOR_BOX_PREVIEW_BASE_HEADER });
-
-        this.annotatedElement.classList.remove(CLASS_ANNOTATION_MODE);
-        this.annotatedElement.classList.remove(CLASS_ANNNOTATION_DRAWING_BACKGROUND);
+    setupHeader(container, header) {
+        super.setupHeader(container, header);
 
-        this.buttonEl.classList.remove(CLASS_ACTIVE);
-
-        this.unbindListeners(); // Disable mode
-        this.emit(CONTROLLER_EVENT.bindDOMListeners);
-    }
+        this.cancelButtonEl = this.getButton(SELECTOR_ANNOTATION_BUTTON_DRAW_CANCEL);
+        this.cancelButtonEl.textContent = this.localized.cancelButton;
 
-    /**
-     * Enables the specified annotation mode
-     *
-     * @inheritdoc
-     * @return {void}
-     */
-    enter() {
-        this.annotatedElement.classList.add(CLASS_ANNOTATION_MODE);
-        this.annotatedElement.classList.add(CLASS_ANNNOTATION_DRAWING_BACKGROUND);
+        this.postButtonEl = this.getButton(SELECTOR_ANNOTATION_BUTTON_DRAW_POST);
 
-        this.buttonEl.classList.add(CLASS_ACTIVE);
+        // TODO(@spramod): Remove '||' string, once doneButton is properly localized within Preview
```

Comment:
Can remove backup text once https://github.com/box/box-content-preview/pull/623 is merged